protocols/P2P.md

```diff
@@ -0,0 +1,17 @@
+# `p2p`
+
+`p2p` is a protocol that defines which peer should be connected to using the PeerId containing its hashed public key.
+
+## Representation Format
+
+### Human-readable
+
+See the PeerId specification.
```

Comment:
How about linking to it here?

Reply:
That will be `https://docs.libp2p.io/concepts/peer-id/`, I guess.